ledger/src/blockstore_db.rs

```diff
@@ -844,5 +843,9 @@ fn get_db_options() -> Options {
     options.create_missing_column_families(true);
     // A good value for this is the number of cores on the machine
     options.increase_parallelism(num_cpus::get() as i32);
+
+    // Set max total wal size to 4G.
+    options.set_max_total_wal_size(4 * 1024 * 1024);
```

Comment:
@sakridge `4 * 1024 * 1024` (= argument) doesn't match to 4G (= comment). Well, is `set_max_total_wal_size` is expected to take the number in kilobytes? Or, should we just use `4 * 1024 * 1024 * 1024`?

Reply:
uh.. yea. thanks

Reply:
Fixed.